packages/harpguru-core/src/utils/ismatch-harpfacematrices/ismatch-harpfacematrices.ts

```diff
@@ -0,0 +1,25 @@
+import { isChromaticHarpFace } from 'harpparts'
+import type { HarpFaceMatrices, HarpFaceMatrix } from 'harpparts'
+
+// TODO: This could use a test file
```

Comment:
It could, but perhaps hold off until we've made a decision about the layoutfacts type.

Reply:
Covered off in https://github.com/js-jslog/harpguru/pull/130. It turned out that this had a major bug in it which didn't come out until the test was written.